cli/src/pcluster/constants.py

```diff
@@ -320,3 +320,5 @@ class Operation(Enum):
 IAM_ROLE_REGEX = "^arn:.*:role/"
 IAM_INSTANCE_PROFILE_REGEX = "^arn:.*:instance-profile/"
 IAM_POLICY_REGEX = "^arn:.*:policy/"
+
+DISABLE_SUDO_ACCESS_FOR_DEFAULT_USER_CONFIG = {"system_info": {"default_user": {"sudo": ["ALL=(ALL) !ALL"]}}}
```

Comment:
This constant represents the cloud-init configuration? If this is the case, let's mention CloudInit in the variable name or add a comment specifying it.